### PR TITLE
Merge pull request #9383 from baker-travis/knob-groups-overflow-not-scrolling

### LINKED ISSUE #9299: addon-knobs tab groups do not support horizontal scrolling on the panel
**Describe the bug**
When there are too many groups for the knobs addon, the tabs flow off the edge of the container. The user is then unable to scroll horizontally to see the hidden tab groups.

**To Reproduce**
1. Create multiple tab groups for knobs so it exceeds the panel width
2. Try and horizontally scroll the tabs

**Expected behavior**
Horizontal scrolling behaviour should be present on the group tabs, much like the tabs for the addons bar above.

**Screenshots**
image

In this screenshot, the addon tabs scroll correctly when there are more addons added, however the tabs for the knobs addon below do not scroll.

### PR DESCRIPTION
**Pull request by baker-travis**
fixes #9299

Issue:

## What I did
Allowed the overflow of the container holding the knob categories.

## How to test
1. Use the official-storybook example project
2. Navigate to Addons -> Knobs -> withKnobs -> tweaks static values organized in groups.
3. Move the panel that contains "Knobs" to the right side of your screen.
4. Observe that you can now scroll the knob categories.

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by ndelangen** [comment]
Thank you @baker-travis !

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
@baker-travis would you be able to resolve the merge conflicts? Would love to merge this in.

**Comment by baker-travis** [comment]
@ndelangen Updated the branch, should be good to go now.

**Comment by ndelangen** [comment]
Thanks!

File: lib/components/src/bar/bar.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ const BarInner = styled.div<{ bgColor: string }>(({ bgColor }) => ({
   flexShrink: 0,
   height: 40,
   backgroundColor: bgColor || '',
+  overflowX: 'scroll',
```
Comment (ndelangen) [inline lib/components/src/bar/bar.tsx]: shouldn't this be auto, btw?

**Review by ndelangen — APPROVED** [review]

File: lib/components/src/bar/bar.tsx (hunk 1 above)
Comment (baker-travis) [reply lib/components/src/bar/bar.tsx]: Yep, thanks for fixing for me!

**Pull request merged**